### DISCUSSION TITLE
German Census 2022 data with 100m resolution – GridViz snippets for 10,786 municipalities

### DISCUSSION THREAD

**Discussion by wahlatlas** [eurostat/gridviz#141] [Show and tell]
You know, my new year's resolution was 100m, and here we are, finally:

### City of Heidelberg – Year of buildings' construction
![Heidelberg_Baujahre](https://github.com/user-attachments/assets/d2780210-428e-4437-869c-ba93f175a084)
Interactive version  
https://wahlatlas.net/experimente/zensus2022/gemeinden/082210000000.html#epoche  

### Municipality Memo showcasing multiple maps on single page
The above is part of a memo page that visualises a wide range of themes, each map being centred on the municipality in question and is available for all 10,786 municipalities of Germany  
https://wahlatlas.net/experimente/zensus2022/

This is still on v2 of GridViz and once I find the time to migrate to v3, things like fullscreen and others will have to be explored.   

You can find the underlying data for your own projects in resolutions of 10km, 1km and 100m at  
https://www.zensus2022.de/EN/News/Hinweis_Zensusatlas_VOE_en.html  

The official implementation chose a different approach, see here for comparison  
https://atlas.zensus2022.de

**Comment by jgaffuri** [comment]
Very nice demo of gridviz ! This dataset is impressive.
Is it going to be officially published soon or can we start sharing it under https://wahlatlas.net/experimente/zensus2022/ URL ?

**Reply by wahlatlas** [reply]
the data is officially published at  
https://www.zensus2022.de/EN/News/Hinweis_Zensusatlas_VOE_en.html  
and all else is and will be my personal project, which you can share freely at  
https://wahlatlas.net/experimente/zensus2022/
